in_toto/in_toto_record.py

```diff
@@ -242,7 +252,8 @@ def main():
     # case the key is encrypted. Something that should not happen in the lib.
     key = None
     if args.key:
-      key = in_toto.util.prompt_import_rsa_key_from_file(args.key)
+      key = in_toto.util.prompt_import_private_key_from_file(args.key,
+          args.key_type)
```

Comment:
Why is it that the `in-toto-record` script does not provide a --prompt command-line option, similar to `in-toto-keygen`?

Do we always want to to prompt for a password when importing a private key with this script?
@SantiagoTorres

Reply:
This doesn't always prompt. It first checks if it can open without a password and if not, then only prompt the user.

Reply:
Oh, the function name `prompt_import_from_private_key()` led me to believe that it does.

This is unexpected because it doesn't quite match the other similar functions:
in_toto.util.generate_and_write_rsa_keypair()
in_toto.util.prompt_generate_and_write_rsa_keypair()

Reply:
I was using the already existing function name. What do you recommend? `try_import_*` ?

Reply:
I guess it's fine to just call it `import_from_private_key()` (or if you want to be verbose, `import_private_or_encrypted_key()`), but it might be better to confirm with @SantiagoTorres.

Reply:
> I guess it's fine to just call it import_from_private_key()

Yeah, I think that `import_private_key_from_file` would do it.